src/fmeval/eval_algorithms/util.py

```diff
@@ -36,10 +36,15 @@
 logger = logging.getLogger(__name__)
 
 
-def get_dataset_configs(data_config: Optional[DataConfig], eval_name: str) -> List[DataConfig]:
-    return (
-        [data_config] if data_config else [DATASET_CONFIGS[dataset_name] for dataset_name in EVAL_DATASETS[eval_name]]
-    )
+def get_dataset_configs(data_config: Optional[Union[DataConfig, List[DataConfig]]], eval_name: str) -> List[DataConfig]:
+    if not data_config:
+        return [DATASET_CONFIGS[dataset_name] for dataset_name in EVAL_DATASETS[eval_name]]
+    elif isinstance(data_config, list):
+        return data_config
+    elif isinstance(data_config, tuple):
```

Comment:
Why are we handling the case where `data_config` is a tuple?

Reply:
Since Python is a dynamic language with type annotations, rather than strictly enforcing types, it seemed like passing in a tuple would be an easy mistake for users to make and falling through to the else to return `[(cfg1, cfg2)]` would be a needlessly annoying/hard-to-debug consequence for that

1. There could be fancier options for more complete `Sequence` support, like checking `hasattr(data_config, "__iter__")` and `__getitem__` (which I believe would include dicts, not ideal) or checking `isinstance(data_config, collections.abc.Sequence)` (which would include strings - also weird)... But idk if you might want to support more sequence-like properties on DataConfig itself one day?
2. We could explicitly check `isinstance(data_config, DataConfig)` rather than having a plain `else`, and throw an error if none of the branches match, but doing `isinstance` on our custom class felt like an even bigger violation of Python's duck typing convention

...So my compromise was just to explicitly support both the basic builtin array-likes but not try to handle other weird `collections` that might crop up more rarely? 🤷‍♂️ But agree it's a little odd.

Reply:
I don't have a strong opinion for or against adding the `elif` branch for handling tuples, but I personally think it makes the code somewhat confusing, since the type annotation for `data_config` doesn't include the `tuple` case. 

I don't think it's very likely for someone to pass in a tuple, as lists are predominantly used. To be honest, I can't even remember the last time I encountered a function accepted a tuple instead of a list.